src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -223,7 +223,7 @@ else if (!Structures.SCHEMATICS_PREFIX.equals(sn.getPrefix()))
         {
             //Look in the folder first
             inputstream = Structure.getStreamFromFolder(MineColonies.proxy.getSchematicsFolder(), structureName);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Close this "InputStream". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2095)